x-pack/test/functional/apps/index_management/data_streams_tab/data_streams_tab.ts

```diff
@@ -23,15 +23,37 @@ export default ({ getPageObjects, getService }: FtrProviderContext) => {
     before(async () => {
       await log.debug('Creating required data stream');
       try {
-        await dataStreams.createDataStream(
-          TEST_DS_NAME,
-          {
-            '@timestamp': {
-              type: 'date',
+        await es.cluster.putComponentTemplate({
```

Comment:
Do we actually need to create a component template for the test setup, or can we include the lifecycle configuration in the index template? (Might simplify the setup and remove a request)